### ISSUE TITLE
Can't use some excelize functions

### ISSUE THREAD

**Issue by Thyrael** [360EntSecGroup-Skylar/excelize#620]
**Description**

Hi, i can't use this function (GetCellCoordonates()) into my golang code. 

```text
excelize.GetCellCoordonates(1,1)
```
And i have this error : GetCellCoordonates not declared by package excelize 

Maybe, i miss something ?

I have the same issue with SetPageLayout() funtion ... 

By the way, i can use all the other function of excelize, but not these

**Comment by xuri** [comment]
Which version of the excelize are you using? The `SetPageLayout` function available since [v2.0.0](https://github.com/360EntSecGroup-Skylar/excelize/releases)). If your package management with Go Modules, please install with

```go
go get github.com/360EntSecGroup-Skylar/excelize/v2
```

**Comment by Thyrael** [comment]
Ok, think i'm tired ... i used old version of excelize. Indeed, with excelize v2.0+, these functions are ok. Thx for your answer @xuri
